corehq/apps/users/login_record.py

```diff
@@ -30,6 +31,10 @@ def _get_raw_fields(cls, username):
     def _get_key(username):
         return f'attempts_{username}'
 
+    @staticmethod
+    def _to_timestamp(raw_ts):
+        return float(raw_ts or EPOCH_TS)
```

Comment:
nit: could remove a bit more duplication by moving `datetime.utcfromtimestamp(ts)` in here too since that's always done with the return value of this function.